src/App.js

```diff
@@ -1,12 +1,14 @@
 import "./App.css";
-import {Routes,Route} from 'react-router-dom'
-
+ import {LandingPage,ProductPage} from "./pages"
+ import{Route,Routes} from 'react-router-dom'
 function App() {
   return (
-    <Routes>
-      
-    </Routes>
+     
+       <Routes>
+         <Route path="/" element={<LandingPage />} />
+         <Route path="/products" element={<ProductPage />} />
```

Comment:
Suggestion : You can make one Route component and use it here to look your app component cleaner as More pages u create more routes will be added into app component.